cadc-access-control/build.gradle

```diff
@@ -24,7 +24,7 @@ dependencies {
     compile 'org.jdom:jdom2:2.+'
     compile 'org.json:json:20160212'
 
-    compile 'org.opencadc:cadc-util:1.+'
+    compile 'org.opencadc:cadc-util:[1.1,)'
     compile 'org.opencadc:cadc-cdp:[1.0,)'
     compile 'org.opencadc:cadc-registry:1.+'
 
```

Comment:
I don't think you need to update the build file here.  No changes so no version update.  It will still get the latest cadc-util.